doc/release-notes-5552.md

```diff
@@ -0,0 +1,6 @@
+Updated RPCs
+--------
+
+- `gobject` and all sub-RPCs are renamed to `govobject`.
+
+The purpose is to force MNs to disable Sentinel.
```

Comment:
It doesn't really matter if they disable it as long as it can't conflict with anything.
```suggestion
The purpose is to prevent conflicts on masternodes that do not remove or disable Sentinel.
```